fully-client-side-calculator/index.js

```diff
@@ -0,0 +1,124 @@
+const screen = document.getElementById("screen");
+const numbers = document.querySelectorAll(".numbers-to-press");
+const operationButtons = document.querySelectorAll(".operation-buttons");
+const equalsButton = document.getElementById("equalsButton");
+const clearButton = document.getElementById("clearButton");
+const decimalButton = document.getElementById("decimalButton");
+let secondValue = "";
+let operand = "";
+let currentValue = 0;
+let operantPressed = false;
+let timesOperandUsed = 0;
+let equalsButtonPurposelyPressed = false;
+let nothingHappened = true;
+const calculator = new Calculator();
+
+function returnValueOfOperation() {
+  if (screen.value === "" || screen.value === ".") return;
```

Comment:
I think screen.value can never be ".", so this second section of the condition is not required.